kubernetes/utils.go

```diff
@@ -71,6 +146,63 @@ func GetNewClientCRD(ctx context.Context, d *plugin.QueryData) (*apiextension.Cl
 	return clientset, err
 }
 
+// GetNewClientCRDRaw :: gets client for querying k8s apis for CustomResourceDefinition
+func GetNewClientCRDRaw(ctx context.Context, cc *connection.ConnectionCache, c *plugin.Connection) (*apiextension.Clientset, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("GetNewClientCRDRaw")
+
+	// have we already created and cached the session?
+	serviceCacheKey := "GetNewClientCRDRaw" //should probably per connection/context keys...
+
+	if cachedData, ok := cc.Get(ctx, serviceCacheKey); ok {
```

Comment:
My linter is flagging this line with:
```
MissingFieldOrMethod: cc.Get undefined (type *connection.ConnectionCache has no field or method Get)
```

Is my linter wrong, or has the function name been changed maybe?